sklearn/utils/estimator_checks.py

```diff
@@ -688,7 +688,11 @@ def check_transformers_unfitted(name, transformer):
     X, y = _boston_subset()
 
     transformer = clone(transformer)
-    assert_raises((AttributeError, ValueError), transformer.transform, X)
+    with assert_raises((AttributeError, ValueError), msg="The unfitted "
+                       "transformer {} does not raise an error when tra"
```

Comment:
Please please please, why would you break the line in the middle of words :upside_down_face: ? Can you fix that everywhere?

Reply:
In order to minimise the number of lines.. sorry :( .. will be done in 5 minutes.

Reply:
@lesteve  Added the changes. Kindly review.

Reply:
Well that was rather creative but there is nothing to be sorry about ;-).